interfaces/builtin/all.go

```diff
@@ -21,72 +21,82 @@ package builtin
 
 import (
 	"github.com/snapcore/snapd/interfaces"
+	"github.com/snapcore/snapd/release"
 )
 
-var allInterfaces = []interfaces.Interface{
-	&BluezInterface{},
-	&BoolFileInterface{},
-	&BrowserSupportInterface{},
-	&ContentInterface{},
-	&DockerInterface{},
-	&DockerSupportInterface{},
-	&FwupdInterface{},
-	&GpioInterface{},
-	&HidrawInterface{},
-	&I2cInterface{},
-	&LocationControlInterface{},
-	&LocationObserveInterface{},
-	&LxdInterface{},
-	&LxdSupportInterface{},
-	&MirInterface{},
-	&ModemManagerInterface{},
-	&MprisInterface{},
-	&NetworkManagerInterface{},
-	&OfonoInterface{},
-	&PppInterface{},
-	&PulseAudioInterface{},
-	&SerialPortInterface{},
-	&UDisks2Interface{},
-	NewAlsaInterface(),
-	NewAvahiObserveInterface(),
-	NewBluetoothControlInterface(),
-	NewCameraInterface(),
-	NewCupsControlInterface(),
-	NewDcdbasControlInterface(),
-	NewFirewallControlInterface(),
-	NewFuseSupportInterface(),
-	NewGsettingsInterface(),
-	NewHardwareObserveInterface(),
-	NewHomeInterface(),
-	NewKernelModuleControlInterface(),
-	NewLibvirtInterface(),
-	NewLocaleControlInterface(),
-	NewLogObserveInterface(),
-	NewMountObserveInterface(),
-	NewNetworkBindInterface(),
-	NewNetworkControlInterface(),
-	NewNetworkInterface(),
-	NewNetworkObserveInterface(),
-	NewNetworkSetupObserveInterface(),
-	NewOpenglInterface(),
-	NewOpticalDriveInterface(),
-	NewProcessControlInterface(),
-	NewRawUsbInterface(),
-	NewRemovableMediaInterface(),
-	NewScreenInhibitControlInterface(),
-	NewShutdownInterface(),
-	NewSnapdControlInterface(),
-	NewSystemObserveInterface(),
-	NewSystemTraceInterface(),
-	NewTimeControlInterface(),
-	NewTimeserverControlInterface(),
-	NewTimezoneControlInterface(),
-	NewTpmInterface(),
-	NewUPowerObserveInterface(),
-	NewUnity7Interface(),
-	NewX11Interface(),
+func initInterfaces() []interfaces.Interface {
```

Comment:
I'm fine with undoing this change. One possible approach would be to fail connection of the interface and removing it from the list of implicitly defined slots.